app/views/homepage/popular_links/_sidebar.html.erb

```diff
@@ -16,8 +16,9 @@
       items: [
         primary_link_button_for(edit_popular_links_path(@latest_popular_links), "Edit popular links") ,
         secondary_button_for(@latest_popular_links, publish_popular_links_path(@latest_popular_links), "Publish"),
-        link_to("Preview (opens in new tab)", preview_homepage_path(@latest_popular_links), rel:"noreferrer noopener", target:"_blank", class: "govuk-link")
-      ]
+        link_to("Preview (opens in new tab)", preview_homepage_path(@latest_popular_links), rel:"noreferrer noopener", target:"_blank", class: "govuk-link"),
```

Comment:
I think this was a side effect of copying the previous line but as this would need to be fixed in multiple places in this file to make it consistent, maybe we can do it as a separate small PR. We can also see if there are similar improvements like this and add them all to the same PR.

Reply:
Ah yes, fair enough. Hopefully it will get sorted in #2274 anyway.